test/jest/turn-off-warnings.js

```diff
@@ -16,6 +16,7 @@ const errorBlacklist = [
   /Warning: Unknown event handler property `%s`./,
   /No metadata harvested from package files, so you will not get app icons./,
   /Warning: Failed prop type: Invalid prop `icon` supplied to `Icon`/,
+  /Warning: Failed %s type: %s%s/,
```

Comment:
Same as comment in `stripes-authority-components` - created a ticket to fix props in tests
https://issues.folio.org/browse/UIPFAUTH-10